src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/probationoffendersearch/ProbationOffenderSearchGatewayTest.kt

```diff
@@ -26,13 +31,57 @@ class ProbationOffenderSearchGatewayTest(
   private val probationOffenderSearchGateway: ProbationOffenderSearchGateway
 ) : DescribeSpec({
   val probationOffenderSearchApiMockServer = ProbationOffenderSearchApiMockServer()
-  val nomsNumber = "xyz4321"
 
   beforeEach {
     probationOffenderSearchApiMockServer.start()
-    probationOffenderSearchApiMockServer.stubPostOffenderSearch(
-      "{\"nomsNumber\": \"$nomsNumber\", \"valid\": true}",
-      """
+    Mockito.reset(hmppsAuthGateway)
+
+    whenever(hmppsAuthGateway.getClientToken("Probation Offender Search")).thenReturn(HmppsAuthMockServer.TOKEN)
+  }
+
+  afterTest {
+    probationOffenderSearchApiMockServer.stop()
+  }
+
+  describe("#getPerson(s)") {
```

Comment:
```suggestion
  describe("#getPersons") {
```

Similar to previous comment!